provisioning/stream/docker/k6/Dockerfile

```diff
@@ -2,10 +2,16 @@ FROM grafana/xk6:latest AS build
 
 RUN xk6 build --with github.com/LeonAdato/xk6-output-statsd
 
-FROM scratch
+FROM alpine:3.20
+
+RUN apk add --no-cache bash
 
 COPY --from=build /xk6/k6 /usr/bin/k6
 
 COPY scripts /scripts
 
-ENTRYPOINT ["k6"]
+COPY provisioning/stream/docker/k6/entrypoint.sh /entrypoint.sh
+
+ENV K6_CLOUD_API_TOKEN=""
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Added entrypoint.sh, you can use k6 cloud.
- It is free for 14 days.
- Limit is  100VUs - but you can set paralel requests to 10 or 100.
- It can be used as statistics output.

![image](https://github.com/user-attachments/assets/ab7d0ae6-bbea-413d-81f3-0bb03632a87d)